Add example: Merge a remote Ignition and other config to the current config (#320)
coreos/fedora-coreos-docs: modules/ROOT/pages/remote-ign.adoc

----

NOTE: The certificate authorities listed here are not automatically added to the host filesystem. They are solely used by Ignition itself when fetching over `https`. If you'd like to also install them on the host filesystem, include them as usual under the `storage.files` array.

> .Retrieving a remote Ignition file via HTTP, also add other config to be merged to the current config

Reviewer: ```suggestion
.Retrieving a remote Ignition file via HTTP and merging it with the current config
```

Author: Thanks for your good suggestion!

----

NOTE: The certificate authorities listed here are not automatically added to the host filesystem. They are solely used by Ignition itself when fetching over `https`. If you'd like to also install them on the host filesystem, include them as usual under the `storage.files` array.

> In some cases, you need to merge a local configuration and one or several remote ones, you can use the `merge` rather than `replace` in a Butane config.

Reviewer: ```suggestion
In some cases, if you need to merge a local configuration and one or several remote ones, you can use the `merge` rather than `replace` in a Butane config.
```
